packages/runtime/src/useCases/consumption/attributes/CreateRepositoryAttribute.ts

```diff
@@ -39,6 +39,20 @@ export class CreateRepositoryAttributeUseCase extends UseCase<CreateRepositoryAt
                 ...request.content
             }
         });
+
+        const queryForRepositoryAttributeDuplicates = flattenObject({
+            content: {
+                "@type": "IdentityAttribute",
+                owner: this.accountController.identity.address.toString(),
+                value: request.content.value
+            }
+        });
+
+        const existingRepositoryAttributes = await this.attributeController.getLocalAttributes(queryForRepositoryAttributeDuplicates);
```

Comment:
I think this introduces a major bug because you don't filter out attributes that have been succeeded. @Milena-Czierlinski & @britsta do you also see a problem with that?

Reply:
Yea, I think you are right, I added a query to only find attributes without a successor.

Reply:
Please also add a test that it is also possible to create a new RepositoryAttribute with a certain `content.value` if the previous RepositoryAttribute with this `content.value` has already been succeeded and is therefore no longer up-to-date.